lib/wovnrb.rb

```diff
@@ -50,11 +51,9 @@ def call(env)
 
         request = Rack::Request.new(env)
         unless request.params['wovn_disable'] == true
+          @store.settings.update_dynamic!(request.params)
```

Comment:
Could you rename into `@store.settings.update_dynamic_settings!` to be consistant with `@store.settings.clear_dynamic_settings!` please?

Reply:
I renamed it.